lighthouse-core/audits/accessibility/object-alt.js

```diff
@@ -15,9 +15,9 @@ const i18n = require('../../lib/i18n/i18n.js');
 
 const UIStrings = {
   /** Title of an accesibility audit that evaluates if all object elements have an alt HTML attribute that describes their contents. This title is descriptive of the successful state and is shown to users when no user action is required. */
-  title: '`<object>` elements have `[alt]` text',
+  title: '`<object>` elements have alt text',
```

Comment:
what about
```suggestion
  title: '`<object>` elements have alternate text',
```
to make the distinction even clearer?

Reply:
Ya sg